project/deps.sc

```diff
@@ -62,12 +62,13 @@ object Deps {
   def slf4jNop                 = ivy"org.slf4j:slf4j-nop:1.8.0-beta4"
   def sparkSql                 = ivy"org.apache.spark::spark-sql:2.4.0"
   def utest                    = ivy"com.lihaoyi::utest:0.7.10"
+  def os_lib                   = ivy"com.lihaoyi::os-lib:0.8.0"
 }
 
 object ScalaVersions {
-  def scala3   = "3.0.1"
-  def cross2_3Version = "2.13.6"
-  def scala213 = "2.13.6"
+  def scala3   = "3.0.2"
```

Comment:
There's a 3.1.x version already, btw.

Reply:
I had a look, as far as I could tell, there is not yet an ammonite version that is published against 3.1.0? It looks like that would get messier than I need... so I will pass on attempting that and wait for the upstream part!

Thankyou very much for the note though :-).